### PR TITLE
doc: add minutes for meeting 3 Jan 2019

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2019-01-03.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+* **Recording**:  https://www.youtube.com/watch?v=2CC7Y9qtj8s
+* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/462
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Liran Tal (@lirantal)
+* Bradley Farias (@bmeck)
+* Nikita Skovoroda
```
Comment (lirantal) [inline meetings/2019-01-03.md]: Add handle `@ChALkeR`

File: meetings/2019-01-03.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* **Recording**:  https://www.youtube.com/watch?v=2CC7Y9qtj8s
+* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/462
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Liran Tal (@lirantal)
+* Bradley Farias (@bmeck)
+* Nikita Skovoroda 
+* David Gonzalez
```
Comment (lirantal) [inline meetings/2019-01-03.md]: Add handle `@dgonzalez`

**Comment by mhdawson** [comment]
@lirantal updated.

**Comment by lirantal** [comment]
LGTM

**Pull request merged**